app/components/articleShow/index.tsx

```diff
@@ -127,6 +127,12 @@ class ArticleShow extends React.PureComponent<IArticleShowProps, {}> {
     dispatch(Actions.votePeerReview(articleId, reviewId));
   };
 
+  private handleUnVotePeerReview = (articleId: number, reviewId: number) => {
+    const { dispatch } = this.props;
+
+    dispatch(Actions.unVotePeerReview(articleId, reviewId));
```

Comment:
You forgot to check the review is mine or not.
you checked it where user deletes a review now.